lib/companies_house/client.rb

```diff
@@ -61,19 +69,33 @@ def make_transaction_id
       SecureRandom.hex(10)
     end
 
+    def company_request(resource,
```

Comment:
I feel a little bit weird about this abstraction - it feels quite odd given that in some cases we pass a `resource` which is also called `:company`, and in some places other things. I think I'd be tempted to stick with the current `request` method.